docker/gatling/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Dockerfile for building gatling image with benchmark Tests
+FROM denvazh/gatling:2.2.1
```

Comment:
We should consider moving to latest version of gatling as i have been using it without any issues

Reply:
OK, Will swap it to latest. That's 2.3.1 right?